src/main/clojure/clara/rules/engine.cljc

```diff
@@ -30,11 +30,19 @@
 (def empty-token (->Token [] {}))
 
 ;; Record indicating the negation existing in the working memory.
-(defrecord NegationResult [gen-rule-name])
```

Comment:
Can you add some comment(s) explaining why we have this divergence between Clojure and ClojureScript here?  This could be pretty puzzling for someone looking at this later without some context.  In particular I suggest mentioning this Clara issue number i.e. https://github.com/cerner/clara-rules/issues/239